ci/run-tests.py

```diff
@@ -31,7 +31,7 @@
     if not os.path.isdir(args.test_dir):
         print('ERROR: invalid test folder: %r' % args.test_dir)
 
-MAX_TRIES = 5
+MAX_TRIES = 1
```

Comment:
This change was made so we keep the original stdout.log and stderr.log. further attempts were always failing in the same way, so we just lost information by having multiple retries. My stance is that if a test is making DF crash, we should fix the test instead of hiding it with retries. To my knowledge, we don't have flaky tests where retries are warranted.

Reply:
crashing tests are now marked as failed and skipped instead of retried